docker/Dockerfile.multiarch

```diff
@@ -0,0 +1,22 @@
+FROM --platform=$BUILDPLATFORM golang:1.16 AS build
+
+RUN apt-get install make
```

Comment:
Should already be included in golang image

```suggestion
```

Reply:
You are correct.  I removed this with no issues.